zebra/rt_netlink.c

```diff
@@ -1325,9 +1325,13 @@ static int netlink_route_multipath(int cmd, struct prefix *p,
 		char buf[NL_PKT_BUF_SIZE];
 	} req;
 
-	struct zebra_ns *zns = zebra_ns_lookup(NS_DEFAULT);
+	struct zebra_ns *zns;
 	struct zebra_vrf *zvrf = vrf_info_lookup(re->vrf_id);
 
+	if (!vrf_is_backend_netns())
```

Comment:
Doesn't the zvrf know it's zns already?  Shouldn't this just be `zns = zvrf->zns`? and we don't need an if statement here?